addons/web/static/src/views/widgets/ribbon/ribbon.scss

```diff
@@ -13,21 +11,8 @@
     }
 
     & span {
-        z-index: 1;
-        position: absolute;
-        display: grid;
-        align-items: center;
         width: 225px;
         height: 48px;
-        padding: 0 44px;
-        box-shadow: 0 5px 10px rgba(0, 0, 0, .1);
-        color: #fff;
-        font: 700 18px/1 'Lato', sans-serif;
-        text-shadow: 0 1px 1px rgba(0, 0, 0, .2);
-        text-transform: uppercase;
-        text-align: center;
-        overflow: hidden;
-        user-select: none;
         &.o_small {
             font-size: 12px;
```

Comment:
`small` class or `<small>` tag ?

Reply:
Those classes are added via JS
<img width="473" alt="Screenshot 2022-09-01 at 16 07 49" src="https://user-images.githubusercontent.com/108661430/187934709-80c936fc-761d-49f3-966b-87e5f20d6fbc.png">

Reply:
Yes. Is it a problem?